R/utilities.R

```diff
@@ -48,6 +48,10 @@ ifNotNull <- function(condition, outputIfNotNull, outputIfNull = NULL) {
 #' See https://github.com/Open-Systems-Pharmacology/OSPSuite-R/issues/476 for details
 #' @import stringr
 encodeUnit <- function(unit) {
+  
+  #TODO maybe we don't even need to load from .NET
+  # dimensionTask <- getNetTask("DimensionTask")

```

Comment:
@Yuri05 If we want to load from .NET we would need to do it here the first time is is being used. But why do we need this in the first place since ` ospsuiteEnv$muSymbol` is already set 
see ospsuite-env.R

# Specifies the symbol used for µ. This will be set by the .NET layer
ospsuiteEnv$muSymbol <- "µ"